Updated docs (#200)
newrelic/nri-flex: docs/basics/configure.md

* If `event_type` is not defined and `name` is, the submitted event type is `name`
  with the `Sample` prefix concatenated.
    - For example, `name: FolderSize` would make Flex to create events named `event_type: FolderSizeSample`.

> * Each `event_type` will be automatically decorated by the infrastructure-agent with a `timestamp` attribute.

Reviewer: ```suggestion
* Each `event_type` is automatically decorated by the infrastructure agent with a `timestamp` attribute.
```

>   Flex can overwrite this attribute, but it's not recomended as it can impact the alerts configured for this `event_type`

Reviewer: ```suggestion
  Flex can overwrite this attribute, but it's not recommended as it can impact the alerts configured for this `event_type`.
```
